metacat-main/src/main/java/com/netflix/metacat/main/services/impl/TableServiceImpl.java

```diff
@@ -458,26 +459,33 @@ public TableDto updateAndReturn(final QualifiedName name, final TableDto tableDt
             handleException(name, ignoreErrorsAfterUpdate, "saveMetadata", e);
         }
 
-        TableDto updatedDto = tableDto;
-        try {
-            updatedDto = get(name,
-                GetTableServiceParameters.builder()
-                    .disableOnReadMetadataIntercetor(false)
-                    .includeInfo(true)
-                    .includeDataMetadata(true)
-                    .includeDefinitionMetadata(true)
-                    .build()).orElse(tableDto);
-        } catch (Exception e) {
-            handleException(name, ignoreErrorsAfterUpdate, "getTable", e);
-        }
+        // ignoreErrorsAfterUpdate is currently set only for iceberg tables
+        if (config.isUpdateIcebergTableAsyncPostEventEnabled() && ignoreErrorsAfterUpdate) {
```

Comment:
Will this apply to all tables?

Reply:
Ah no, just iceberg tables. ignoreErrorsAfterUpdate was added for a similar purpose and is set to true only for Iceberg. We're re-using that flag here.